client/my-sites/site-settings/theme-enhancements.jsx

```diff
@@ -76,7 +79,46 @@ class ThemeEnhancements extends Component {
 		);
 	}
 
-	renderInfiniteScrollSettings() {
+	renderSimpleSiteInfiniteScrollSettings() {
+		const { customizeUrl, translate } = this.props;
+		return (
+			<FormFieldset>
+				<FormLegend>{ translate( 'Infinite Scroll' ) }</FormLegend>
+
+				<div className="theme-enhancements__info-link-container site-settings__info-link-container">
+					<InfoPopover position="left">
+						{ translate( 'Control how additional posts are loaded.' ) }
+						<br />
+						<ExternalLink
+							href="https://support.wordpress.com/infinite-scroll/"
+							icon
+							target="_blank"
+						>
+							{ translate( 'Learn more' ) }
+						</ExternalLink>
+					</InfoPopover>
+				</div>
+
+				{ this.renderToggle(
+					'infinite_scroll',
+					false,
+					translate( 'Load posts as you scroll. Disable to show a clickable button to load posts.' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Check to load posts as you scroll. Uncheck to show clickable button to load posts' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).